07_Train_With_AML_Pipeline.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
    "source": [
     "# Publish a Training Pipeline\n",
-    "In this notebook, we will show how to automate the training/retraining of model using HyperDrive and registering best model. Once this training pipeline is published/created, it provides a REST endpoint which can be called to run this pipeline without using the Azure Machine Learning Service SDK.\n",
+    "In this notebook, we will show how to automate the training/retraining of model using HyperDrive and registering best model. Once this training pipeline is published/created, it provides a rest endpoint which can be called to run this pipeline or we can create schedule to run this pipeline.\n",
```

Comment:
The published happy path deliberately does not create a scheduled pipeline. Doing so can leads to user problems and unexpected user costs. This would be fine for the instrument branch.